src/apic-extension/azext_apic_extension/custom.py

```diff
@@ -202,43 +202,47 @@ def register_apic(cmd, api_location, resource_group, service_name, environment_i
             contact = info.get('contact')
             if contact:
                 extracted_api_contact_email = contact.get('email')
                 extracted_api_contact_name = contact.get('name')
                 extracted_api_contact_url = contact.get('url')
                 contacts = [{'email': extracted_api_contact_email, 'name': extracted_api_contact_name, 'url': extracted_api_contact_url}]
-            else:
-                contacts = None
 
             # Create API - Get the license details from info in spec
+            extracted_api_license = None
             licenseDetails = info.get('license')
             if licenseDetails:
                 extracted_api_license_identifier = licenseDetails.get('identifier')
                 extracted_api_license_name = licenseDetails.get('name')
                 extracted_api_license_url = licenseDetails.get('url')
                 extracted_api_license = {'identifier': extracted_api_license_identifier, 'name': extracted_api_license_name, 'url': extracted_api_license_url}
-            else:
-                extracted_api_license = None
 
             # Create API - Get the terms of service from info in spec
+            extracted_api_terms_of_service = {'url': None}
             extracted_api_terms_of_service_value = info.get('termsOfService')
             if extracted_api_terms_of_service_value:
                 extracted_api_terms_of_service = {'url': extracted_api_terms_of_service_value}
-            else:
-                extracted_api_terms_of_service = {'url': None}
 
-            # Create API - Get the external documentation from info in spec
-            extracted_api_external_documentation = None
-            external_documentation = info.get('externalDocumentation')
+            # Create API - Get the external documentation in spec
+            cnt_docs = 0
+            extracted_api_external_documentation = []
+            external_documentation = data.get('externalDocs')
             if external_documentation:
                 extracted_api_external_documentation_description = external_documentation.get('description')
-                extracted_api_external_documentation_title = external_documentation.get('title')
                 extracted_api_external_documentation_url = external_documentation.get('url')
-                extracted_api_external_documentation = {'description': extracted_api_external_documentation_description, 'title': extracted_api_external_documentation_title, 'url': extracted_api_external_documentation_url}
-            else:
-                extracted_api_external_documentation = None
-
+                cnt_docs += 1
+                extracted_api_external_documentation.append({'description': extracted_api_external_documentation_description, 'title': f'Doc_{cnt_docs}', 'url': extracted_api_external_documentation_url})
+
+            # Create API - Get the external documentation from the tags in the spec
+            tags = data.get('tags')
```

Comment:
Good findings on the external doc in tags. Since the external document is for the API itself per Azure Portal. Let don't add external docs for the tags.